homeassistant/components/sonos/media_player.py

```diff
@@ -239,8 +239,8 @@ def available(self) -> bool:
         """Return if the media_player is available."""
         return (
             self.speaker.available
-            and self.speaker.sonos_group_entities
-            and self.media.playback_status
+            and self.speaker.sonos_group_entities is not None
```

Comment:
```suggestion
            and bool(self.speaker.sonos_group_entities)
```
`speaker.sonos_group_entities` is a `list[str]` and will thus never be `None`.